include/graaflib/algorithm/graph_isomorphism_util.h

```diff
@@ -0,0 +1,165 @@
+#pragma once
+
+#include <graaflib/graph.h>
+#include <graaflib/types.h>
+#include <cstddef>
+#include <memory>
+#include <optional>
+
+// Delete later
+#include <iostream>
+
+/**
+HELPER CLASSES
+*/
+
+namespace graaf::algorithm::vf2{
+
+struct predecessors_and_successors_of_vertex {
```

Comment:
I understand it is verbose, but for someone who has an idea of how the algorithm works, this name is pretty descriptive and says exactly what the struct is for. I would like to keep this as the name, if you would allow it.